README.md

```diff
@@ -1092,15 +1092,23 @@ If you need to customize the conversion of markdown to HTML you can supply the `
 import { Markdown, defaultSanitizerOptions } from 'rendition/dist/extra/Markdown';
 ```
 
+Generated html inherits styles from rendition components. i.e an anchor will be
+rendered as a `Link` component.
+
+Html inside the markdown is sanitized and stripped of any inline js and css. so
+they will always be rendered.
+
+Components can be overridden by using `componentOverrides` prop.
 
 [View story source](https://github.com/balena-io-modules/rendition/blob/master/src/extra/Markdown/story.js)
 
 #### Props
 
-| Name   | Type   | Default   | Required   | Description   |
-| ------ | ------ | --------- | ---------- | ------------- |
-| `children`  | `string` | - | ✓ | The markdown source that should be rendered |
-| `sanitizerOptions` | [`sanitizeHtml.IOptions`](https://github.com/apostrophecms/sanitize-html#how-to-use) | (See `extra/utils/defaultSanitizerOptions`) | - | Specifies the options used when sanitizing the generated HTML. |
+| Name                 | Type                                                                 | Default | Required | Description                                                    |
+| -------------------- | -------------------------------------------------------------------- | ------- | -------- | -------------------------------------------------------------- |
+| `children`           | `string`                                                             | -       | ✓        | The markdown source that should be rendered                    |
+| `sanitizerOptions`   | [`Object`](https://github.com/syntax-tree/hast-util-sanitize#schema) |         | -        | Specifies the options used when sanitizing the generated HTML. |
+| `componentOverrides` | `Object`                                                             | -       | -        | Object specifying component Overrides. ex. `{ p: Txt.p }`      |
```

Comment:
Do you need this currently? If not, I'd rather omit it for now, and we can add it later on if truly necessary.

Reply:
@sradevski yes, I need it in landr.